mesonbuild/backend/ninjabackend.py

```diff
@@ -1429,7 +1429,7 @@ def generate_cs_compile_rule(self, compiler, outfile):
         rule = 'rule %s_COMPILER\n' % compiler.get_language()
         invoc = ' '.join([ninja_quote(i) for i in compiler.get_exelist()])
         command = ' command = %s $ARGS $in\n' % invoc
-        description = ' description = Compiling cs target $out.\n'
+        description = ' description = Compiling C# target $out.\n'
```

Comment:
You seems to be right here. I wanted to put the actual unicode for the sharp character. Can we escape thoses with Ninja?